sdk/nodejs/api/example/todoapp/README.md

```diff
@@ -0,0 +1,106 @@
+# todo-react
```

Comment:
I don't think we need to put `todoapp` in the SDK

Reply:
I'm using the todoapp to test the querybuilder. I can move it away though

Reply:
Todoapp has been removed